lib/livebook_web/live/session_live/app_teams_live.ex

```diff
@@ -341,7 +345,13 @@ defmodule LivebookWeb.SessionLive.AppTeamsLive do
     ~H"""
     <div class="flex gap-4 sm:gap-12 border border-gray-200 rounded-lg p-4">
       <.labeled_text label="Slug">
-        /<%= @app_deployment.slug %>
+        <.link
+          href={@deployment_group.url <> ~p"/apps/#{@app_deployment.slug}"}
+          target="_blank"
+          class="text-blue-600 font-medium"
+        >
+          /<%= @app_deployment.slug %>
+        </.link>
```

Comment:
The group may not have a URL set

```suggestion
        <%= if @deployment_group.url do %>
          <.link
            href={@deployment_group.url <> ~p"/apps/#{@app_deployment.slug}"}
            target="_blank"
            class="text-blue-600 font-medium"
          >
            /<%= @app_deployment.slug %>
          </.link>
        <% else %>
          <span>
            /<%= @app_deployment.slug %>
          </span>
        <% end %>
```